AndrewTehan/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require_relative '../my_hash'
+require 'pry'
+
+RSpec.describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  describe '#[]=' do
+    context 'when key is missing' do
+      it 'adds new key-value pair' do
+        expect(subject[1] = :a).to eq(:a)
+        expect(subject[1]).to eq(:a)
+      end
+    end
+
+    context 'when key exists' do
+      before do
+        subject[1] = 1
+      end
```

Comment:
if you have one line of block, you can easily replace it with `{ subject[1] = 1 }`